includes/class-wc-tracker.php

```diff
@@ -621,6 +624,60 @@ private static function get_order_dates() {
 
 		return array_merge( $min_max, $processing_min_max );
 	}
+
+	/**
+	 * Search a specific post for text content.
+	 *
+	 * @param integer $post_id The id of the post to search.
+	 * @param string  $text    The text to search for.
+	 * @return string 'Yes' if post contains $text (otherwise 'No').
+	 */
+	public static function post_contains_text( $post_id, $text ) {
+		global $wpdb;
+
+		$result = $wpdb->get_var(
+			$wpdb->prepare(
+				"
+				SELECT COUNT( * ) FROM {$wpdb->prefix}_posts
```

Comment:
I believe this should be `{$wpdb->prefix}posts`, like line 593. When I run this method in `wpcli` I get an error message about table `wp__posts` not existing.

Reply:
Now fixed in 0b79a65829d03ccf8a9d09774be685862c2bf3a5, thanks for spotting :)